### DISCUSSION TITLE
First-child on padding-top not work

### DISCUSSION THREAD

**Discussion by shiqocred** [tailwindlabs/tailwindcss#13916] [Help]
**What version of Tailwind CSS are you using?**

For example: v2.0.4

**What build tool (or framework if it abstracts the build tool) are you using?**

Next.js 14.1.2

**What version of Node.js are you using?**

v20.0.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Describe your issue**

```
<div className="flex flex-col text-xs md:text-sm py-4 before:w-[1px] before:content-[''] before:absolute before:left-3 md:before:left-5 before:bg-gray-300 before:h-full before:top-0 relative z-0 first:pt-8 before:dark:bg-gray-700 last:pb-0">
   ...
</div>
```

`first-child:` is not work when i use on `BubbleChat` like

```
<div className="w-full">
     {chats.map((item) => (
        <BubbleChat key={item.id} {...item} />
      ))}
</div>
```

**Comment by wongjn** [comment]
Seems to be working in [this Tailwind Play](https://play.tailwindcss.com/9Sb6WQisSP). Could you provide a fuller reproduction of when it doesn't work please?
